lib/rollbax/notifier.ex

```diff
@@ -44,4 +44,10 @@ defmodule Rollbax.Notifier do
     %{level: Keyword.get(config, :level, :error),
       metadata: Keyword.get(config, :metadata, [])}
   end
+
+  defp ts_to_unix({{_yy, _mm, _dd} = date, {h, m, s, _ms}}) do
+    now = :calendar.datetime_to_gregorian_seconds({date, {h, m, s}})
```

Comment:
It's not `now`, rather `event_time` (`log_time`?), but having `@unix_epoch` we probably don't even need to name it. :bowtie: